app/views/tariffs/_form.html.erb

```diff
@@ -1,7 +1,11 @@
 <%= form_for :tariff, url: tariffs_path do |f| %>
   <p>
-    <%= f.label :name, 'Назва тарифу' %>
-    <%= f.text_field(:name, { class: 'form-control' }) %>
+    <%= f.label :format_photo_id, 'Група' %>
+   <%= f.collection_select :tariff_group_id, TariffGroup.all, :id, :name,  {label: "Група тарифів", prompt: "Виберіть групу тарифів"}  %>
+  </p>
+  <p>
+    <%= f.label :tariff_format_id, 'Формат' %>
+    <%= f.collection_select :format_photo_id, FormatPhoto.all, :id, :name,  {label: "Формат фото", prompt: "Виберіть формат фото"}  %>
```

Comment:
@romigos please move `FormatPhoto.all` to value of controller(like `format_photos = FormatPhoto.all`) and use it
also please do for TariffGroup.all

Reply:
i do not understand all is working